src/styles/header.scss

```diff
@@ -0,0 +1,65 @@
+.header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  margin-left: 50px;
+}
+
+.nav {
+  margin-right: 50px;
+
+  &__list {
+    display: flex;
+    align-items: center;
+  }
+
+  &__item {
+    font-size: 12px;
+    font-weight: 500;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+
+    &:not(:first-child) {
+      margin-left: 20px;
+    }
+  }
+
+  &__link {
+    display: inline-block;
+    text-decoration: none;
+    color: $black;
+    text-transform: uppercase;
+    line-height: 60px;
+    transition: background-color 0.2s;
```

Comment:
Also, as I see in some cases transition time is 0.2s and 0.3s, time should be the same for all transitions, you can create a variable to avoid this from happening

Reply:
done